src/pages/ProfilePage.js

```diff
@@ -21,7 +43,20 @@ export default function ProfilePage({
                 src="https://images.unsplash.com/photo-1487174244970-cd18784bb4a4?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1173&q=80"
               />
             </PictureCropper>
-            <ProfileName>Creepster</ProfileName>
+            <Input
+              type="text"
+              aria-label="Field name"
+              onChange={onChange}
+              onKeyDown={onKeyDown}
+              value={editingValue}
+              onBlur={onBlur}
+              maxLength="10"
+            />
+            <img
+              src={editPen}
+              alt="You can edit the name next to the pen"
+              height="25px"
+            />
```

Comment:
It seems that your pen icon has no further function. Wouldn´t it be better to click on the pen to edit your profile name? Just think about it ;)